rm-datasource/src/main/java/io/seata/rm/datasource/exec/InsertExecutor.java

```diff
@@ -359,4 +388,12 @@ protected boolean checkPkValues(List<Object> pkValues) {
         return pkValues;
     }
 
+    private Map<String, ColumnMeta> checkPrimaryKeyMap() {
```

Comment:
How about remove this method? Because multi primary key already check in TableMeta.getPrimaryKeyMap()